File: Web/src/app/models/access-control/employee-field.ts
export class EmployeeField {
  AccessControlRoleName: string;
  EntityCode: number;
  EntityName: string;
  UnionName: string;
  EmployeeActive: Boolean;

  constructor(param: any) {
    this.AccessControlRoleName = param.AccessControlRoleName,
    this.EntityCode = this.EntityCode,
    this.EntityName = param.EntityName
    this.UnionName = param.UnionName
    this.EmployeeActive = param.EmployeeActive
  }
}

File: Api/Timesheet/Entities/EmployeeField.cs
using Microsoft.EntityFrameworkCore;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Timesheet.Entities
{
    [Keyless]
    public class EmployeeField
    {
        public string AccessControlRoleName { get; set; }
        public string EmployeeName { get; set; }
        public string EmployeeEmail { get; set; }
        public string UnionName { get; set; }
        public string FieldName { get; set; }
        public string EntityCode { get; set; }
        public string EntityName { get; set; }
        public Boolean AcceptNotices { get; set; }
        public Boolean SendNotices { get; set; }
        public Boolean EmployeeActive { get; set; }
    }
}
